dyn_em/module_big_step_utilities_em.F

```diff
@@ -2545,154 +2553,172 @@ SUBROUTINE w_damp( rw_tend, max_vert_cfl,max_horiz_cfl, &
 !  horizontal motion.  These values are returned via the max_vert_cfl and
 !  max_horiz_cfl variables.  (Added by T. Hutchinson, WSI, 3/5/2007)
 !
+!-----
+!
+!  W_damp modified to be more flexible with IEVA capability. Two things changed.
+!  First, the value of the W-CFL where damping is turned on can now be set in the namelist
+!  using "w_crit_cfl".  Second, the code has been modified to report two levels of
+!  W-CFL information.  Previously, both damping and W-CFL information was
+!  printed at a W-CFL == 1.2.  This new capability does not need IEVA on to run.
+!
+!  We keep the old level reporting (for consistency), but use "w_flag_cfl" to flag 
+!  and print information from those points.  "w_flag_cfl" is set here in w_damp.  
+!  A second level of print occurs, which is useful for when we run
+!  IEVA, because many points can run with W-CFL > 1.2.  We now print this
+!  second level separately, as these points might make the model blow up.
+!  and where we now want Rayleigh damping turned on.  The default value "w_crit_cfl" 
+!  which replaced "w_beta" in the old code, is set in the namelist/Registery
+!  turn on w_damp at 1.2 consistent with the WRFV3/4 (before this) code.
+!
+!  Summary
+!  -------
+!  W_damp will write out CFL information when the vertical courant number is > w_flag_cfl,
+!  and W_damp will also turn on Rayleigh damping if vertical courant number > w_crit_cfl.
+!
+!  Typical settings for non-IEVA use:  w_crit_cfl = 1.2
+!  Typical settings for     IEVA use:  w_crit_cfl = 2.0
+!
+!  I also commented out a lot of code put in 8-13 years ago for timing assuming that
+!  this is no longer needed....there is some pretty hacky stuff.  HTH.
+!
+!      (Added by L. Wicker, NSSL, 5/4/2020)
+!
 !</DESCRIPTION>
 
    itf=MIN(ite,ide-1)
    jtf=MIN(jte,jde-1)
 
-   some = 0
```

Comment:
Lou,
The initialization of some1 and some2 to zero is similar to data statement, where it is only done once. Since you have some1 and some2 on the RHS down below, go ahead and initialize them to zero here.

Reply:
Yep, I see those new inits, thanks